src/Models/Constants/Interfaces.ts

```diff
@@ -141,10 +142,26 @@ export interface IConsumeCompositeCardProps extends ICardBaseProps {
 }
 
 export interface IAuthService {
-    login: () => void;
-    getToken: (
-        tokenFor?: 'azureManagement' | 'adx' | 'storage' | 'powerBI'
-    ) => Promise<string>;
+    /** the unique object id of the logged in user */
+    userObjectId?: string;
```

Comment:
do we have to expose teh value directly or can we expose a setter and getter? Feels weird for a service to let other components set a property directly.

Reply:
I added getter instead